tools/authors/add.php

```diff
@@ -54,34 +54,46 @@
         // insert into the database
         if ($author_id) {
             // edit existing author
-            $result = mysqli_query(DPDatabase::get_connection(), sprintf("
+            $result = DPDatabase::query(sprintf("
```

Comment:
Per the [best practices guide](https://www.pgdp.net/wiki/DP_Code_Best_Practices#Building_SQL), please make this:
```
$sql = sprintf("
    ...
);
$result = DPDatabase::query($sql);
```
Ditto elsewhere.